unittests/data/type_traits.hpp

```diff
@@ -421,6 +415,7 @@ namespace no{
     typedef void(*function_t)();
     typedef void (some_struct_t::*member_function_t)();
     typedef int int_t;
+    typedef const int& const_int_ref_t;
```

Comment:
Moved this from `yes` to `no`. `is_const` should return `False` for `int const&`, see [this question](http://stackoverflow.com/q/24029723/241631) I answered on SO a while back.